### DISCUSSION TITLE
Allow real value of private nullable

### DISCUSSION THREAD

**Discussion by wimwinterberg** [nette/php-generator#167] [General]
Since the latest version 4.1.6 there is indeed a BC with isNullable. Is it possible to add way to also retrieve the real nullable value?

Now when I want a property to have a default value of null I use 'new Literal('null')'. So there is a difference between having value null and new Literal('null')

Thank you!

**Comment by dg** [comment]
You are probably looking for the `setInitialized()` and `isInitialized()` methods.
